docs/cluster-configuration.md

```diff
@@ -260,10 +260,16 @@ Bandwidth limit of each migrationi, the value is in quantity of bytes per second
 
 ### completionTimeoutPerGiB
 
-The migration will be canceled if it has not completed in this time, in seconds per GiB of memory. For example, a
-virtual machine instance with 6GiB memory will timeout if it has not completed migration in 4800 seconds. If the
-Migration Method is BlockMigration, the size of the migrating disks is included in the calculation. The format is a
-number.
+If a migrating VM is big and busy, while the connection to the destination node 
+is slow, migration may never converge. The completion timeout is calculated
+based on completionTimeoutPerGiB times the size of the guest (both RAM and 
+migrated disks, if any). For example, with completionTimeoutPerGiB set to 800,
+a virtual machine instance with 6GiB memory will timeout if it has not
```

Comment:
Question, is this also going to affect workload upgrades when `spec.workloadUpdateStrategy.workloadUpdateMethods` contains `LiveMigrate`?

See: https://issues.redhat.com/browse/CNV-35196

Reply:
I am trying (with the help of @vladikr ) to document the present situation because the current docs do not make much sense to me.  I'm not an expert of the inner workings. FWIW, I bet this applies to all migrations, regardless of what triggers them. https://kubevirt.io/api-reference/master/definitions.html#_v1_virtualmachineinstancemigrationspec has no way to make a specific migration behave different than the cluster-wide configuration.


https://github.com/kubevirt/hyperconverged-cluster-operator/blob/main/docs/cluster-configuration.md#enable-eventual-launcher-updates-by-default

Reply:
@tiraboschi from what I gather, currently there is no differentiation between eviction- and upgrade triggered migration.